src/mozmlops/templates/README.md

```diff
@@ -68,6 +68,20 @@ You can see an example Dockerfile [here](Dockerfile-metaflow) in this templates
 
 Please note that if your step should use our NVIDIA GPUs, it cannot use a docker image at present, as the Outerbounds `@nvidia` decorator does not interoperate with docker image specification. In this case, please use the `@pypi` and `@conda` decorators for dependencies as documented above.
 
+ ## Using Vertex AI with Model Orchestration Flows
+
+These flows work in Vertex AI Workbench (which some teams at Mozilla use for cloud-based notebook experimentation: it's Google's answer to AWS Sagemaker.) You can put `some_flow.py` into your workbench and run it from a terminal tab in the workbench.
+
+You can also incorporate flow runs into your Jupyter notebooks. [This Runner syntax](https://docs.metaflow.org/api/runner) facilitates calling a flow from a notebook. Alternatively, Outerbounds' [metaflow-card-notebook](https://github.com/outerbounds/metaflow-card-notebook) module facilitates running notebooks from a DAG.
```

Comment:
Does it make sense to add an example jupyter notebook in this repo too? No is a fine answer

Reply:
I don't _think_ so, because the way to run a flow from a notebook is just a few lines, and exemplified pretty handily at the top of the Runner syntax doc.